module-services/service-cellular/connection-manager/ConnectionManager.cpp

```diff
@@ -34,6 +35,7 @@ void ConnectionManager::onTimerTick()
         if (minutesOnlineElapsed.count() >= connectedPeriod.count()) {
             minutesOnlineElapsed = static_cast<std::chrono::minutes>(0);
             onlinePeriod         = false;
+            cellular->holdMinimumCpuFrequency();
             cellular->disconnectFromNetwork();
```

Comment:
Have you tested whether after disconnecting from the network, min CPU freqency is released and the modem goes into sleep mode?

Reply:
```
2438729 ms DEBUG [ServiceCellular] ATCommon.cpp:cmdLog:47: [AT]: >AT+CFUN=4<, >OK<
2438729 ms INFO  [ServiceCellular] ATCommon.cpp:cmdLog:52: [AT] > OK
2438852 ms DEBUG [CellularMux] ServiceCellular.cpp:operator():170: Notifications callback called with 22 data bytes
2438852 ms DEBUG [CellularMux] ServiceCellular.cpp:operator():173: Data: +QIND: "csq",99,99
2438852 ms DEBUG [CellularMux] ServiceCellular.cpp:identifyNotification:900: Notification:: +QIND: "csq",99,99
2438852 ms INFO  [CellularMux] CellularUrcHandler.cpp:Handle:124: Invalid csq - ignore
2438853 ms DEBUG [CellularMux] ServiceCellular.cpp:operator():170: Notifications callback called with 12 data bytes
2438853 ms DEBUG [CellularMux] ServiceCellular.cpp:operator():173: Data: +CREG: 0
2438853 ms DEBUG [CellularMux] ServiceCellular.cpp:identifyNotification:900: Notification:: +CREG: 0
2438853 ms INFO  [CellularMux] CellularUrcHandler.cpp:Handle:42: Network status - NotRegistered, access technology Unknown
2440159 ms DEBUG [CellularMux] ServiceCellular.cpp:operator():170: Notifications callback called with 26 data bytes
2440160 ms DEBUG [CellularMux] ServiceCellular.cpp:operator():173: Data: +QIND: "act","UNKNOWN"
2440160 ms DEBUG [CellularMux] ServiceCellular.cpp:identifyNotification:900: Notification:: +QIND: "act","UNKNOWN"
2441701 ms INFO  [SysMgrService] RT1051LPM.cpp:SetCpuFrequency:97: CPU frequency changed to 264000000
2442704 ms INFO  [SysMgrService] RT1051LPM.cpp:SetCpuFrequency:97: CPU frequency changed to 132000000
2450006 ms INFO  [SysMgrService] RT1051LPM.cpp:SetCpuFrequency:97: CPU frequency changed to 66000000
2451004 ms INFO  [SysMgrService] RT1051LPM.cpp:SetCpuFrequency:97: CPU frequency changed to 24000000
2452003 ms INFO  [SysMgrService] RT1051LPM.cpp:SetCpuFrequency:97: CPU frequency changed to 12000000
```